lib/providers/app_lifecycle_notifier_provider.dart

```diff
@@ -16,6 +17,7 @@ class AppLifecycleNotifier extends Notifier<void> {
 
   void onAppResumed(BuildContext context) {
```

Comment:
What would happen if user logs in, closes the app and opens the app back again in under one minute? Based on this user would need to log in again - which is not the desired behaviour